bundles/org.openhab.ui.basic/src/main/resources/snippets/icon_material.html

```diff
@@ -1 +1 @@
-<spam class="material-icons md-oh">%icon_name%</spam>
\ No newline at end of file
+<spam class="material-icons">%icon_name%</spam>
```

Comment:
Not related to your PR, but shouldn't this be `<span>` and not `<spam>`...?

Reply:
Oups, right.

Reply:
Now fixed. Thank you for your finding.